src/components/common/YourCompany/YourCompany.vue

```diff
@@ -207,30 +207,11 @@
 
     <v-divider class="mx-4 my-1" />
 
-    <!-- Business Start Date (changes, conversions and firm corrections only) -->
-    <template v-if="isChangeRegFiling || isFirmConversionFiling || isFirmCorrectionFiling">
```

Comment:
I don't see this v-if in the new component you created. Isn't it still needed for this section?

Reply:
Yes I will add those v-ifs in

Reply:
Hey, please let the reviewer resolve the conversations they started.

I use open conversations as a bookmark of code to re-review.